### PR TITLE
chore: add missing requirement

### PR DESCRIPTION
**Pull request by kaustavb12**
## Description

The [latest commit](https://github.com/overhangio/tutor/commit/0f2229b181c584c760442be6f063ea77f120a41e) to nighly is breaking deployments because it's using the [packaging](https://github.com/overhangio/tutor/blob/master/tutor/commands/upgrade/common.py) package, which is not included in the requirements.

This PR adds that as a requirement.

## Other information

Private Ref : [BB-9216](https://tasks.opencraft.com/browse/BB-9216)

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -38,6 +38,8 @@ oauthlib==3.2.2
     # via
     #   kubernetes
     #   requests-oauthlib
+packaging==24.1
+    # via -r requirements/base.in
```
Comment (pomegranited) [inline requirements/base.txt:42]: Consistency nit: the other `requirements/*.txt` use `packaging==23.2`

Comment (pomegranited) [reply requirements/base.txt:42]: And don't we need it in `tutor@master` too?

**Review by pomegranited — APPROVED** [review]
:+1: Thanks for submitting this PR @kaustavb12 :) Saved me some time.

- [x] I tested this by deploying to OpenCraft's tagging-preview sandbox (private link [Grove pipeline](https://gitlab.com/opencraft/ops/grove-stage-digitalocean/-/pipelines/1471833020)) using `tutor@nightly` + this added requirement
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A
- [x] <strike>User-facing strings are extracted for translation</strike> N/A

File: requirements/base.txt (hunk 1 above)
Comment (kaustavb12) [reply requirements/base.txt:42]: Thanks @pomegranited for the review.

Synced the requirements from base.txt to the other *.txt files. It's consistent now.

> And don't we need it in tutor@master too?

The commit which used this package is only present in `nightly` right now. I am unsure about the process here, but I am assuming that changes in `nigtly` will be syced to `master` at some point ?

Comment (regisb) [reply requirements/base.txt:42]: I think it was a mistake to merge that PR in nightly only. Can you change your PR to target master please?

Comment (kaustavb12) [reply requirements/base.txt:42]: @regisb PR rebased to target master.

**Review by regisb — APPROVED** [review]
thanks for the lightning fast PR!

**Pull request merged**